api/app/weather_models/process_grib.py

```diff
@@ -35,11 +36,11 @@ class ModelRunInfo():
     """
 
     def __init__(self):
-        self.model_enum: ModelEnum = None
-        self.projection: ProjectionEnum = None
-        self.model_run_timestamp = None
-        self.prediction_timestamp = None
-        self.variable_name = None
+        self.model_enum: Union(ModelEnum, None) = None
```

Comment:
`Optional[ModelEnum] = None`
`Optional[ProjectionEnum] = None`
`Optional[datetime] = None`
`Optional[str] = None`

Reply:
Thanks - missed that one.